includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -327,7 +327,7 @@ public function get_subtext_messages( $stripe_method_status ) {
 			$text            = __( 'Pending activation', 'woocommerce-gateway-stripe' );
 			$tooltip_content = sprintf(
 				/* translators: %1: Payment method name */
-				esc_attr__( '%1$s won\'t be visible to your customers until you provide the required information. Follow the instructions Stripe has sent to your e-mail.', 'woocommerce-gateway-stripe' ),
+				esc_attr__( '%1$s won\'t be visible to your customers until you provide the required information. Follow the instructions Stripe has sent to your e-mail address.', 'woocommerce-gateway-stripe' ),
```

Comment:
I ensured the messaging is consistent between the "old" and the "new" settings, so that we have fewer strings to translate floating around.